### PR TITLE
GEODE-8326: remove 5 minutes wait to get stack dump

### PR DESCRIPTION
**Pull request by pivotal-eshu**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by pivotal-eshu** [comment]
Thanks for pointing out the default timeout time. I have updated the test so that stack dump can be generated if failed again.

File: geode-core/src/distributedTest/java/org/apache/geode/internal/cache/partitioned/fixed/FixedPartitioningWithTransactionDistributedTest.java [hunk 1]
```diff
@@ -238,7 +238,7 @@ private void forceClientMetadataUpdate(Region region) {
     ClientMetadataService clientMetadataService =
         ((InternalCache) clientCacheRule.getClientCache()).getClientMetadataService();
     clientMetadataService.scheduleGetPRMetaData((InternalRegion) region, true);
-    await().atMost(5, MINUTES).until(clientMetadataService::isMetadataStable);
+    await().atMost(5, HOURS).until(clientMetadataService::isMetadataStable);
```
Comment (agingade) [inline geode-core/src/distributedTest/java/org/apache/geode/internal/cache/partitioned/fixed/FixedPartitioningWithTransactionDistributedTest.java:241]: 5 hours is a long time...Will this change going to be merged to develop? Can the debug be tried with local/private build?

Comment (pivotal-eshu) [reply geode-core/src/distributedTest/java/org/apache/geode/internal/cache/partitioned/fixed/FixedPartitioningWithTransactionDistributedTest.java:241]: It is not reproducible in private/local build. We only need to have it to fail once, once the stack dump is triggered, we can/should change it back to default timeout.

Comment (onichols-pivotal) [reply geode-core/src/distributedTest/java/org/apache/geode/internal/cache/partitioned/fixed/FixedPartitioningWithTransactionDistributedTest.java:241]: seems like this experiment could have been conducted in the PR pipeline, shouldn't have needed to merge to develop

**Pull request merged**